pkg/actors/targets/workflow/orchestrator/orchestrator.go

```diff
@@ -83,38 +90,40 @@ func (o *orchestrator) InvokeTimer(ctx context.Context, reminder *actorapi.Remin
 	return errors.New("timers are not implemented")
 }
 
-func (o *orchestrator) Completed() bool {
-	return o.completed.Load()
-}
-
-func (o *orchestrator) InvokeStream(ctx context.Context, req *internalsv1pb.InternalInvokeRequest, stream chan<- *internalsv1pb.InternalInvokeResponse) error {
+func (o *orchestrator) InvokeStream(ctx context.Context, req *internalsv1pb.InternalInvokeRequest, stream func(*internalsv1pb.InternalInvokeResponse) (bool, error)) error {
 	o.wg.Add(1)
 	defer o.wg.Done()
-	return o.handleStream(ctx, req, stream)
-}
 
-// DeactivateActor implements actors.InternalActor
-func (o *orchestrator) Deactivate(ctx context.Context) error {
-	if !o.closed.CompareAndSwap(false, true) {
-		return nil
+	unlock, err := o.lock.ContextLock(ctx)
```

Comment:
this is new, no? there wasn't a requirement before that did not allow to InvokeStream on an actor multiple times concurrently